src/Analyzers/MSTest.Analyzers/TestMethodShouldBeValidAnalyzer.cs

```diff
@@ -72,49 +64,49 @@ public override void Initialize(AnalysisContext context)
 
         if (methodSymbol.MethodKind != MethodKind.Ordinary)
         {
-            context.ReportDiagnostic(methodSymbol.CreateDiagnostic(OrdinaryRule, methodSymbol.Name));
+            context.ReportDiagnostic(methodSymbol.CreateDiagnostic(ValidTestMethodSignatureRule, DiagnosticDescriptorHelper.CannotFixProperties, methodSymbol.Name));
 
             // Do not check the other criteria, users should fix the method kind first.
             return;
         }
 
         if (methodSymbol.IsAbstract)
         {
-            context.ReportDiagnostic(methodSymbol.CreateDiagnostic(NotAbstractRule, methodSymbol.Name));
+            context.ReportDiagnostic(methodSymbol.CreateDiagnostic(ValidTestMethodSignatureRule, methodSymbol.Name));
```

Comment:
You need to edit the code to avoid reporting X times the issue in case of multiple conflicts as we no longer have a specific message.